models/retinanet/builder.py

```diff
@@ -417,8 +427,8 @@ def get_retinanet_neck(data):
             bias=X.var(name="P4_lateral_bias", init=X.zero_init()),
             name="P4_lateral"
         )
-        p5_clip = mx.sym.Crop(*[p5_up, p4_la], name="P4_clip")
-        p4 = mx.sym.ElementWiseSum(*[p5_clip, p4_la], name="P4_sum")
+        p5_clip = mx.sym.slice_like(*[p5_up, p4_la], name="P4_clip")
```

Comment:
Should be `p5_clip = mx.sym.slice_like(p5_up, p4_la, name="P4_clip")`

Reply:
@xchani